ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -225,6 +226,17 @@ _get_mobilenet_ssd_pp_threshold (bounding_boxes * bdata)
   return bdata->mobilenet_ssd_pp.threshold;
 }
 
+/** @brief Mathematic inverse of sigmoid function, aka logit */
+static float logit(float x)
+{
+  if (x < 0)
+    x = 0;
+  if (x > 1)
+    x = 1;
+
+  return log(x / (1.0 - x));
```

Comment:
Could you please try to avoid division-by-zero? (unless dbz is intended)

Reply:
I verified this implementation behaves correctly, with regard to infinity handling in the log().
So I believe it's safe but some exception might be raised in log() depending on the implementation.
Hence, I modified the logit() implementation to avoid infinity handling in log(), and returns directly +- INFINITY for input values out of range.
The simple scalar comparison operation should not be an issue while handling INFINITY in the rest of the code.